package.json

```diff
@@ -33,6 +33,7 @@
     "@elastic/eslint-config-kibana": "link:../../packages/eslint-config-kibana",
     "@elastic/eslint-import-resolver-kibana": "link:../../packages/kbn-eslint-import-resolver-kibana",
     "@elastic/eui": "6.10.7",
+    "@elastic/charts": "^18.2.0",
```

Comment:
Do we plan to upgrade `@elastic/charts` or try `react vis`?

Reply:
adding `@elastic/charts` to "dependencies" works. no need to move to `react-vis`

Reply:
Cool, that's great!